app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb

```diff
@@ -0,0 +1,16 @@
+<p>If you already have a degree, or equivalent qualification, you’re ready for postgraduate primary or secondary initial teacher training.</p>
+
```

Comment:
@EmmaFrith I changed this from:
`<p>If you already have a degree, or equivalent qualification, you’re ready for postgraduate initial teacher training.</p>

 <p>For postgraduate primary or secondary teacher training you need the following or equivalents:</p>`.

Can I get your thoughts. Think this is the final thing. Trying to make sure I'm super clear that this applies to just primary and secondary, even though we did this in Step 1

Reply:
Do we define anywhere what is considered equivalent to a degree?

Reply:
We don't, but that would require a new page to explain!

Reply:
[Level 6](https://www.gov.uk/what-different-qualification-levels-mean/list-of-qualification-levels#level-6)?